modules/installation-aws-regions-with-no-ami.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-regions-with-no-ami_{context}"]
+= AWS regions without a published {op-system} AMI
+
+You can deploy an {product-title} cluster to Amazon Web Services (AWS) regions
+without native support for an {op-system-first} AMI or the AWS SDK. When
+deploying to an unknown region, you must specify the custom endpoints that are
+required to access AWS APIs.
+
+[NOTE]
+====
+Deploying to an AWS Government region or specialized public commercial region
+are common use cases where a region would require a custom {op-system} AMI.
+====
+
+You must define the following AWS service endpoints if you provide an unknown
+region.
+
+- `ec2`
+- `elasticloadbalancing`
+- `iam`
+- `route53`
+- `s3`
+- `tagging`
+
+// installer
+
+A custom AMI is not available to select from the terminal during cluster
+creation because it is not published. However, you can install to this region by
+configuring the custom AMI in the `install-config.yaml` file. The installation
+program copies the AMI to the user account automatically and encrypts it using
+the KMS key of the default user account, allowing the custom AMI to follow the
+same process workflow as published {op-system} AMIs.
```

Comment:
- the installer terminal prompts only show regions that have Red Hat published AMIs. But you can install to other AWS commercial regions by setting those regions in install-config.yaml . Here  the installer will import the AMI to the user's account before using it to boot machines.

- The installer no longer encrypts the AMI when importing rather it creates control plane machines with encrypted EBS volumes using default KMS key or user specified.